rapid7_insightvm/komand_rapid7_insightvm/util/resource_helper.py

```diff
@@ -38,15 +38,17 @@ class ResourceHelper(object):
     """
 
     # Currently only handling the most common requests exceptions more can be added as needed
-    Request_exceptions = {
-        requests.HTTPError: '',
-        requests.ConnectionError: '',
-        requests.Timeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
+    REQUEST_EXCEPTIONS = {
+        requests.HTTPError: ' If this issue persists contact support for assistance.',
+        requests.ConnectionError: 'Unable to connect to IVM consul.'
+                                  ' If this issue persists contact support for assistance',
+        requests.Timeout: 'Ensure proper network connectivity between the InsightConnect orchestrator and the InsightVM console',
         requests.ConnectTimeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
         requests.ReadTimeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
-        requests.TooManyRedirects: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
-        'Unhandled exception': 'contact support for assistance'
+        requests.TooManyRedirects: 'Ensure proper network connectivity between the orchestrator and the IVM consul'
```

Comment:
```suggestion
        requests.TooManyRedirects: 'Ensure proper network connectivity between the InsightConnect orchestrator and the IVM console'
```
There are a lot of typos in the above lines as well - those need to be fixed

Reply:
Some of these are duplicates as well and should be turned into constants.